tasks/main.yml

```diff
@@ -95,17 +96,107 @@
     with_items:
       - main.cf
       - master.cf
+      - dynamicmaps.cf
+      - cleanup-headers
+      - ldap-map.cf
+    notify:
+      - restart postfix
+    tags: postfix
+
+  - name: Configure local aliases
+    become: True
+    when: mailserver_system_owner != False
+    template:
+      src: postfix/aliases.j2
+      dest: /etc/aliases
+      owner: root
+      group: root
+      mode: 0644
+    notify: newalias
+
+  - name: Configure postfix (store local domain)
+    become: True
+    register: dommap
+    copy:
+      content: "{{ mailserver_domain }} OK"
+      dest: /etc/postfix/local_domains
+      owner: root
+      group: root
+      mode: 0644
+    notify:
+      - restart postfix
+    tags: postfix
+
+  - name: Configure postfix (generate local domain map)
+    become: True
+    when: dommap.changed
+    shell: postmap /etc/postfix/local_domains
+    args:
+      chdir: /etc/postfix
+    notify:
+      - restart postfix
+    tags: postfix
+
+  - name: Configure postfix (MySQL config)
+    become: True
+    when: mailserver_config_method == "mysql"
+    template: src='postfix/{{item}}.j2' dest='/etc/postfix/{{item}}' owner=root group=root mode=0644
```

Comment:
code style

Reply:
Hmm, the file is already long enough as-is. Can we agree to allow the shorthand for `template` `with_items`?